### PR TITLE
OKTA-657288 Add appName note in SCIM doc

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Provide instructions to obtain the appName variable for SCIM docs
- **Is this PR related to a Monolith release?** No

### Preview:
https://65c16dc7af2b9b00801e5a18--reverent-murdock-829d24.netlify.app/docs/guides/scim-provisioning-integration-prepare/main/#authentication

### Resolves:

* [OKTA-657288](https://oktainc.atlassian.net/browse/OKTA-657288)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/ef236dc4d0d35226ef6902722c56d9203d33be4b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [62](https://okta.acrolinx.cloud/api/v1/checking/scorecards/044d2557-c95f-44c6-a8f0-2b4b0f11cc05) | 64  | 44  | 43  | 96  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9fe211e9-7ec1-4f66-b697-8ce2359b0b57?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/cafa8d4e7d65b39f49a69945c39573d8911ff0be/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c52bb75f-2185-4991-a17b-6d4e60239f42) | 85  | 52  | 57  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.bbf9f196-452e-49af-b280-c579fb238377?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/3e3d144cef08b801478f7a5633a3d158e02c1713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b31e7e19-7b35-4607-af24-d00133f4dc91) | 87  | 52  | 62  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.5269b13d-38dc-44b0-8aa3-dde5447ac135?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/a321355c1b596c0eae6b552531ba8ef1b17ea142/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1602e2c0-a4e7-4029-8511-877500d71a72) | 87  | 52  | 62  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.09474d17-2dd3-452e-b134-1c6a3b4173cb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/b4f7b9d7273a3ed1055363bd4cdb93a4bcffd588/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0e739902-0d2f-4eda-99a9-2b152311f005) | 90  | 51  | 61  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.a8f00d93-31b9-44a0-b1ac-c3c06bf6b5b7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by ykrestyanov-okta — APPROVED** [review]
lgtm

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md [hunk 1]
```diff
@@ -93,42 +101,42 @@ Okta requires that your SCIM implementation be able to store the following four
...
+If your integration supports user attributes beyond those four base attributes, you can add support for more attributes to your SCIM API. Sometimes, you might need to configure Okta to map non-standard user attributes into the user profile for your application. See [Check the attributes and corresponding mappings](/docs/guides/scim-provisioning-integration-connect/main/#check-the-attributes-and-corresponding-mappings), or [Add custom attributes to an Okta user profile](https://help.okta.com/okta_help.htm?id=ext_Directory_Profile_Editor_Tasks) in the Okta product documentation.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:106]:
```suggestion
If your integration supports user attributes beyond those four base attributes, you can add support for more attributes to your SCIM API. Sometimes, you might need to configure Okta to map non-standard user attributes into the user profile for your app. See [Check the attributes and corresponding mappings](/docs/guides/scim-provisioning-integration-connect/main/#check-the-attributes-and-corresponding-mappings), or [Add custom attributes to an Okta user profile](https://help.okta.com/okta_help.htm?id=ext_Directory_Profile_Editor_Tasks) in the Okta product documentation.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md [hunk 2]
```diff
@@ -138,44 +146,43 @@ Sometimes it isn't feasible for your cloud-based application to natively support
 
 ## Provision to on-premises applications
 
-This provisioning guide targets cloud-based applications, but Okta does have a solution for on-premise applications. See [Provision on-premises provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) for details about the Okta agent-based provisioning solution.
+This provisioning guide targets cloud-based applications. For on-premise applications, see [Provision on-premises provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) for details about the Okta agent-based provisioning solution.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:149]:
```suggestion
This provisioning guide targets cloud-based apps. For on-premise apps, see [Provision on-premises provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) for details about the Okta agent-based provisioning solution.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md [hunk 3]
```diff
@@ -138,44 +146,43 @@ Sometimes it isn't feasible for your cloud-based application to natively support
...
     - [Okta SCIM 2.0 Spec Test JSON](/standards/SCIM/SCIMFiles/Okta-SCIM-20-SPEC-Test.json)
     - [Okta SCIM 1.1 Spec Test JSON](/standards/SCIM/SCIMFiles/Okta-SCIM-11-SPEC-Test.json)
 
-1. If you are new to Runscope, your dashboard displays sample Runscope tutorials. Click **Skip Tutorial**.
-1. You should now see a screen titled **API Tests**.
-1. In the lower left of your screen, click **Import Test**.
-1. A new screen appears, titled **Import Tests into &#x2026;**
+1. If you're new to Runscope, your dashboard displays sample Runscope tutorials. Click **Skip Tutorial**.
+1. You should now see a page titled **API Tests**.
+1. In the lower left of your page, click **Import Test**.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:170]:
```suggestion
1. In the lower left of your page, click **Import Test**. A new page appears, titled **Import Tests into &#x2026;**.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md [hunk 4]
```diff
@@ -138,44 +146,43 @@ Sometimes it isn't feasible for your cloud-based application to natively support
...
+1. A new page appears, titled **Import Tests into &#x2026;**
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:171]:
```suggestion

**Review by barbaravo-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md [hunk 5]
```diff
@@ -138,44 +146,42 @@ Sometimes it isn't feasible for your cloud-based application to natively support
...
+1. Download the Okta SCIM specification test file that corresponds to the version of SCIM you set up on your server. Use this file to import the Okta SCIM test suite into Runscope.
 
     - [Okta SCIM 2.0 Spec Test JSON](/standards/SCIM/SCIMFiles/Okta-SCIM-20-SPEC-Test.json)
     - [Okta SCIM 1.1 Spec Test JSON](/standards/SCIM/SCIMFiles/Okta-SCIM-11-SPEC-Test.json)
 
-1. If you are new to Runscope, your dashboard displays sample Runscope tutorials. Click **Skip Tutorial**.
-1. You should now see a screen titled **API Tests**.
-1. In the lower left of your screen, click **Import Test**.
-1. A new screen appears, titled **Import Tests into &#x2026;**
+1. If you're new to Runscope, your dashboard displays sample Runscope tutorials. Click **Skip Tutorial**.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:168]:
```suggestion
1. If you're new to Runscope, your dashboard displays sample Runscope tutorials. Click **Skip Tutorial**. A page titled **API Tests** appears.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md [hunk 6]
```diff
@@ -138,44 +146,42 @@ Sometimes it isn't feasible for your cloud-based application to natively support
...
+1. You should now see a page titled **API Tests**.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:169]:
```suggestion
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/4fc8b0275b81691e3fb6618006ca1ad19fca3fc6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/adbb0b98-a17f-4e61-add4-1d911dcec1a0) | 90  | 50  | 61  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3d06488a-b36f-4358-8b17-f35d48010e74?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md](https://github.com/okta/okta-developer-docs/blob/7a5a4b9eed1c2a4a89b034410eb14f09ff40dbe2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-provisioning-integration-prepare%2Fmain%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0b102886-f1d3-459f-b61a-40da1bcfbc9b) | 90  | 50  | 61  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.1cf3651e-5575-470e-8fc3-33b4d459cae7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**